### PR TITLE
Update readthedocs for SVM catalog generation

### PR DESCRIPTION
**Pull request by dulude**
**Relevant JIRA Tickets and Git Issues**
- Ticket [HLA-785](https://jira.stsci.edu/browse/HLA-785)
- Issue #1386 

**Summary**
- Implemented suggestions from Ticket [HLA-785](https://jira.stsci.edu/browse/HLA-785):
   - Table 1 currently has columns 'Inner aperture', 'Outer aperture'.  Change these column names to Aper1 and Aper2, for consistency with the catalog headers and with the HSC.
   - Provide threshold values (e.g. 5-sigma) for point catalogs for each detector, if there is a simple prescription used. 
   - Add text to describe aperture photometry: "_These catalogs provide aperture photometry in the ABMAG system and are calibrated using the photometric zeropoints corresponding to an 'infinite' aperture. To convert to total magnitudes, aperture corrections must be applied to account for flux falling outside of the selected aperture. For details, see Whitmore et al., 2016 AJ, 151, 134W._ "
- Corrected now-obsolete paths listed in section 1.2 of the "catalog generation" documentation page
- Reworked portions of section 1.2 of the "catalog generation" documentation page
- A bit of code smuggling:
  - runmultihap.py: added new optional command-line input 'input_custom_pars_file' that allows users to specify the name of a custom-tuned parameter file to use. NOTE: This feature has existed in hapmultisequencer.py for some time. This update simply makes it available to users who run runmultihap.py from the command-line.

**Note**
When I rebuilt the documentation HTML pages with my changes to catalog_generation.rst, all bulleted text throughout the entirety of the drizzlepac documentation lost their bullets. I don't think it's anything I did when making the above updates, as the same thing happens when I rebuilt the documentation from the current version of master.

### CODE REVIEW

File: doc/source/catalog_generation.rst [hunk 1]
```diff
@@ -133,10 +138,17 @@ with the associated updates, are ultimately chosen as the images to use.
...
-/drizzlepac/pars/hap_pars/default_parameters/<instrument>/<detector>/.
+the general situation, but users can tune these values to optimize for their own data.
+
+To this end, users can adjust
+parameter values in the <instrument>_<detector>_catalog_generation_all.json files in the following path:
+/drizzlepac/pars/hap_pars/svm_parameters/<instrument>/<detector>/. Alternatively, a safer way for users to tune
+configuration settings is to generate custom parameter files with the drizzlepac/haputils/generate_custom_svm_mvm_param_file.py
+tool, adjust values as needed and use them in their single visit mosaic pipeline runs utilizing the '-c' command-line
+argument or the 'input_custom_pars_file' optional input argument when executing `hapsequencer.run_hap_processing()`
+
```
Comment (mdlpstsci) [inline doc/source/catalog_generation.rst:149]: I would clarify the discussion here a bit.  Explain how to use **runsinglehap** with the custom parameter file from the command line, as well as **hapsequencer.run_hap_processing** from within a Python session.

File: drizzlepac/runmultihap.py [hunk 2]
```diff
@@ -54,6 +54,11 @@ def perform(input_filename, **kwargs):
     debug : Boolean
         display all tracebacks, and debug information?
 
+    input_custom_pars_file : str, optional
+        Represents a fully specified input filename of a configuration JSON file which has been
+        customized for specialized processing. This file should contain ALL the input parameters necessary
+        for processing. If not specified, default configuration parameter values will be used.
+
```
Comment (mdlpstsci) [inline drizzlepac/runmultihap.py:61]: Good catch to notice support for custom parameter files was missing from MVM processing.

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Just a small change in some comments, and this PR is good to go.

File: doc/source/catalog_generation.rst (hunk 1 above)
Comment (dulude) [reply doc/source/catalog_generation.rst:153]: Suggestions implemented in commit [3032dae](https://github.com/spacetelescope/drizzlepac/pull/1400/commits/3032dae544530eea7ccd2655c394ce3199d560b5)

**Review by mdlpstsci — APPROVED** [review]
Looks fine with the updates.

**Pull request merged**